spring-boot-project/spring-boot-docs/src/docs/asciidoc/web/spring-security.adoc

```diff
@@ -223,7 +223,23 @@ Again, the same properties are applicable for both servlet and reactive applicat
 
 Alternatively, you can define your own `OpaqueTokenIntrospector` bean for servlet applications or a `ReactiveOpaqueTokenIntrospector` for reactive applications.
 
+To enable audience validation, set the `spring.security.oauth2.resourceserver.jwt.audiences` property in your Spring Boot application 
+configuration file. This property specifies the expected value(s) of the aud claim in JWTs.
 
+For example, to expect the JWTs to contain an aud claim with the value `my-audience`, you can add the following line to your 
+application.properties file:
+
+[source,yaml,indent=0,subs="verbatim",configprops,configblocks]
+----
+configprop:spring.security.oauth2.resourceserver.jwt.audiences[]
```

Comment:
I have moved it. I am afraid that the syntax might not be right. Thank you for your patience.

Reply:
Sorry, it's still not right and you've moved it to line 229 rather than updating 226 as I requested above. Please don't make any more changes. I'll take things from here.

Reply:
I did a lot of troubleshooting and the build is successful now.

May I commit my changes, please? @wilkinsona 

<b>This is what it would look like.</b>
----------------------------------------------------------------------------------------------------------------

To enable audience validation, set the `configprop:spring.security.oauth2.resourceserver.jwt.audiences[]` property in your Spring Boot application
configuration file. This property specifies the expected value(s) of the aud claim in JWTs.

For example, to expect the JWTs to contain an aud claim with the value `my-audience`, you can add the following line to your 
application.properties file:

[source,yaml,indent=0,subs="verbatim",configprops,configblocks]
----
	spring:
	  security:
	    oauth2:
	      resourceserver:
	        jwt:
	          audiences:
	            - "my-audience"
----
---------------------------------------------------------------------------------------------------------------------

Reply:
That looks better. Please go ahead.